src/core/CPU.cpp

```diff
@@ -1056,17 +1056,26 @@ Blaze::Cycles Blaze::CPU::executeJSR(AddressingMode mode) {
 };
 
 Blaze::Cycles Blaze::CPU::executeLDA(AddressingMode mode) {
-	// TODO
+	// TODO TO CHECK
+	Address val = loadOperand(mode);
+	A = val;
```

Comment:
Actually, if the accumulator is in 8-bit mode, `LDA` (and any other instruction that modifies the accumulator) only modifies the lower byte and leaves the high 8 bits alone. I also coded this wrong in `ADC`; I only noticed it now after re-reading the manual. I'll go back and fix that and check any other instructions that modify the accumulator, as well.